tycho-indexer/src/extractor/protocol_extractor.rs

```diff
@@ -2680,25 +2837,50 @@ mod test_serial_db {
                 ]),
                 component_balances: HashMap::from([
                     ("pc_1".to_string(), HashMap::from([
-                        (Bytes::from_str("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48").unwrap(), ComponentBalance {
-                            token: Bytes::from_str("0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48").unwrap(),
+                        (Bytes::from_str(USDC_ADDRESS).unwrap(), ComponentBalance {
+                            token: Bytes::from_str(USDC_ADDRESS).unwrap(),
                             balance: Bytes::from("0x00000064"),
                             balance_float: 100.0,
                             modify_tx: Bytes::from_str("0x0000000000000000000000000000000000000000000000000000000000007532").unwrap(),
                             component_id: "pc_1".to_string(),
                         }),
-                        (Bytes::from_str("0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2").unwrap(), ComponentBalance {
-                            token: Bytes::from_str("0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2").unwrap(),
+                        (Bytes::from_str(WETH_ADDRESS).unwrap(), ComponentBalance {
+                            token: Bytes::from_str(WETH_ADDRESS).unwrap(),
                             balance: Bytes::from("0x00000001"),
                             balance_float: 1.0,
                             modify_tx: Bytes::from_str("0x0000000000000000000000000000000000000000000000000000000000000000").unwrap(),
                             component_id: "pc_1".to_string(),
                         }),
                     ])),
                 ]),
+                account_balances: HashMap::from([
```

Comment:
Can we also add a test that verifies that our "merge" is correct?
I'm worried about the case where we get 2 AccountUpdates for the same account on different transactions.